project/Dependencies.scala

```diff
@@ -13,6 +13,7 @@ object Dependencies {
   val AkkaHttpVersion = "10.0.9"
   val ScalaVersion = "2.11.11"
   val AkkaPersistenceCassandraVersion = "0.54"
+  val AkkaPersistenceJdbcVersion = "3.0.0"
```

Comment:
Not yet released. Next version supporting Akka 2.5.4 and Slick 3.2.1 will be 3.0.0